File: pyg_user_web/src/main/java/com/pyg/user/controller/LoginController.java
package com.pyg.user.controller;

import org.springframework.security.core.context.SecurityContextHolder;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.HashMap;
import java.util.Map;

@RestController
public class LoginController {
    @RequestMapping("/getLoginUser")
    public Map getLoginUser(){
        String name = SecurityContextHolder.getContext().getAuthentication().getName();
        Map result = new HashMap();
        result.put("username",name);
        return  result;
    }
}

File: pyg_seckill_web/src/main/webapp/js/controller/loginController.js
//控制层
app.controller('loginController' ,function($scope,loginService){


    //获取当前你登录名
    $scope.getLoginUser = function () {
        loginService.getLoginUser().success(
            function (response) {
                $scope.username = response.username;
            }
        );
    }
});